### PR TITLE
Picotable Mass Actions (SH-351)

### PR DESCRIPTION
**Pull request by Pikkupomo**
Add Mass actions to Contacts, Orders and Products

Refs SH-351

### CODE REVIEW

**Comment by Pikkupomo** [comment]
I will add tests to these new features next week.

File: shuup/admin/modules/contacts/views/mass_edit.py [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+
+
+class ContactGroupMassEditView(MassEditMixin, FormView):
+    title = _("Mass Edit: Contact Groups")
+    form_class = GroupMassEditForm
+
+    def form_valid(self, form):
+        groups = form.cleaned_data.get("contact_group", [])
+        for group in groups:
+            for id in self.ids:
```
Comment (gurch101) [inline shuup/admin/modules/contacts/views/mass_edit.py]: should be able to do group.members.add(*self.ids)

**Comment by tulimaki** [comment]
Missing tests.

File: shuup/admin/static_src/picotable/picotable.js [hunk 2]
```diff
@@ -391,11 +392,23 @@ const Picotable = (function(m, storage) {
...
+        var massActions = (ctrl.vm.data() ? ctrl.vm.data().massActions : null);
+
         // Build body
         var isPick = !!ctrl.vm.pickId();
         var rows = Util.map(data.items, function(item) {
-            return m("tr", {key: "item-" + item._id}, Util.map(data.columns, function(col) {
-                var content = item[col.id] || "";
+            return m("tr", {key: "item-" + item._id}, Util.map(data.columns, function(col, idx) {
+                if (idx == 0 && massActions.length) {
+                    var content = m("input[type=checkbox]", {
```
Comment (Pikkupomo) [inline shuup/admin/static_src/picotable/picotable.js]: Didn't have time to make select all for this version.

**Comment by Pikkupomo** [comment]
Added https://shuupteam.atlassian.net/browse/SH-407 based on the comment by @gurch101

File: shuup/admin/static_src/picotable/picotable.js (hunk 2 above)
Comment (Pikkupomo) [inline shuup/admin/static_src/picotable/picotable.js]: https://shuupteam.atlassian.net/browse/SH-408 created this for class

**Pull request merged**